playground/src/main.ml

```diff
@@ -101,7 +101,7 @@ let handle_output (o : Toplevel_api.exec_result) =
   let output_elements =
     List.filter_map
       (fun (c, o) -> render_output c o)
-      [ (cyan, o.stdout); (red, o.stderr); (red, o.caml_ppf) ]
+      [ (cyan, o.stdout); (white, o.stderr); (white, o.caml_ppf) ]
```

Comment:
Note: I don't know this part of the codebase.

I'm curious now, where does the playground render the REPL response? Does it render via `o.caml_ppf` or via `o.stderr`?

`stderr` being red makes sense if it only shows the errors.